xhttp/request_metrics.go

```diff
@@ -38,21 +38,30 @@ func (rm *requestMetrics) statusCode(reqURI string, statusCode int) string {
 }
 
 var (
-	keyForHTTPStats = []string{"http", "request"}
+	keyForHTTPReqPerf       = []string{"http", "request", "perf"}
```

Comment:
@haykbaluyan adding here `perf`,`successful`, `failed`. We will need to adjust the dashboard